letsencrypt/client/network2.py

```diff
@@ -534,6 +530,8 @@ def fetch_chain(self, certr):
         """
         if certr.cert_chain_uri is not None:
             return self._get_cert(certr.cert_chain_uri)[1]
+        else:
```

Comment:
Why not just use the default return None?  This decision was suggested/implemented elsewhere in the code.